teradatalabs/centos6-java8-oracle-maprclient/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Teradata
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM teradatalabs/centos6-java8-oracle
+MAINTAINER Teradata Docker Team <docker@teradata.com>
+
+ADD files/maprtech.repo /etc/yum.repos.d/maprtech.repo
+RUN yum update -y \
```

Comment:
I don't think there's a default set of command that needs to be executed for this container. 
The first step is to configure the client - https://github.com/Teradata/presto/pull/528/files#diff-d5fa118c169436bb50bbd774dc84d440R51
And this needs to be done only once, not every time you log into the container.
I can have CMD as bash, but it currently works without that.
`mapr-client $docker run -it mapr-client `
`[root@aec1a9c03a4c /]# `
So not sure if we need ENTRYPOINT/CMD